mlir/test/Dialect/SparseTensor/no_fold_into_consumer.mlir

```diff
@@ -24,7 +24,6 @@ module {
   // CHECK:       arith.constant
   // CHECK:       tensor.empty()
   // CHECK:       linalg.generic
```

Comment:
rename test file 

fold_into_producer

(instead of no-fold_into_consumer : we originally avoided a bug in the folding into consumer, which was fixed, but you also made it a fold into producer now!